### PR TITLE
Support from .X import Y

### PR DESCRIPTION
**Pull request by diorcety**
Work as-is with setup.py mechanism, for command line invocation a package argument can be provided in order to specify the current package of the "compiled" python file

### CODE REVIEW

File: pythran/run.py [hunk 1]
```diff
@@ -82,6 +84,9 @@ def run():
                         'generation',
                         default=list())
 
+    parser.add_argument('-C', dest='package', type=str,
```
Comment (serge-sans-paille) [inline pythran/run.py]: That's a strange option name; Maybe only accept long option?

File: pythran/toolchain.py [hunk 2]
```diff
@@ -451,6 +454,8 @@ def compile_pythranfile(file_path, output_file=None, module_name=None,
 
     # Add compiled module path to search for imported modules
     sys.path.append(os.path.dirname(file_path))
+    if package is not None:
+        sys.path.append(os.getcwd())
```
Comment (serge-sans-paille) [inline pythran/toolchain.py]: I don't get this one.

**Comment by serge-sans-paille** [comment]
You should add a few test case in pythran/tests/test_user_defined_import.py

File: pythran/run.py (hunk 1 above)
Comment (diorcety) [reply pythran/run.py]: sure

File: pythran/toolchain.py (hunk 2 above)
Comment (diorcety) [reply pythran/toolchain.py]: This make the setup.py hook work nice :)

Comment (diorcety) [reply pythran/toolchain.py]: This need improvements I know

Comment (serge-sans-paille) [reply pythran/toolchain.py]: yeah :-)

**Comment by serge-sans-paille** [comment]
Obsoleted by #913

**Pull request closed without merging**